content/docs/learn/integrations.md

```diff
@@ -78,6 +78,16 @@ The main point of contact between Ktor and Arrow is in
 If you're using kotlinx.serialization, you need no further changes other than
 importing the serializers with `@UseSerializers`.
 
+If you want to use Arrow Core types directly as your request or response models, you will need to include the `ArrowModule` in your serializers module:
+
+```kotlin
```

Comment:
Please, do not use ```kotlin here (or in the rest of the file), as it would trigger Knit, but these blocks are not complete Kotlin code.

Reply:
I've removed the `kotlin` tags from the code blocks, and also from the pre-existing blocks for consistency.

They weren't being picked up by knit in the build (at least a `./gradlew knit build` passed) as there were no knit patterns in either file, but should one be added at a later it's better to avoid surprises.